spec/support/user_helper.rb

```diff
@@ -3,7 +3,7 @@ def login(email)
   click_link 'Войти'
   fill_in :email, with: email
   fill_in :password, with: 12345
-  click_button 'Log In'
+  click_button 'Войти'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.